sqlserver/tests/test_connection.py

```diff
@@ -166,7 +166,8 @@ def test_will_fail_for_wrong_parameters_in_the_connection_string(instance_minima
 @pytest.mark.parametrize(
     'host, port, expected_host',
     [
-        pytest.param('1.2.3.4', '22', '1.2.3.4,22', id='if port provided as a config option, it should be recognized'),
+        pytest.param('1.2.3.4', 22, '1.2.3.4,22', id='if port provided as a config option as an int, it should be recognized'),
+        pytest.param('1.2.3.4', 22, '1.2.3.4,22', id='if port provided as a config option as a string, it should be recognized'),
```

Comment:
That's a duplicate of the line above except for the `id`, right? Shouldn't the second one do `'22'`?